tests/components/mystrom/test_config_flow.py

```diff
@@ -0,0 +1,131 @@
+"""Test the myStrom config flow."""
+from unittest.mock import AsyncMock, patch
+
+from pymystrom.exceptions import MyStromConnectionError
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.mystrom.const import DOMAIN
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+DEVICE_MAC = "6001940376EB"
+DEVICE_NAME = "myStrom Device"
+
```

Comment:
You can import `DEVICE_MAC` from `conftest` module.
You shouldn't need to declare `DEVICE_NAME`